components/styles.js

```diff
@@ -73,8 +73,8 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
-    left: 13,
-    top: 13,
+    left: (DEVICE_WIDTH / 14) - 14,
```

Comment:
Actually, what was the reasoning for changing it to absolute positioning in your last PR?